frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -18,39 +20,54 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
   });
 
   const handleFormSubmit = useCallback(
-    (event_: React.BaseSyntheticEvent): void => {
-      void handleSubmit(onSubmit)(event_);
+    (event: React.BaseSyntheticEvent): void => {
+      void handleSubmit(onSubmit)(event);
     },
     [handleSubmit, onSubmit],
   );
 
   return (
-    <>
-      <h3>Sign Up</h3>
-      <form onSubmit={handleFormSubmit}>
-        <p>
-          <Input
-            type="text"
-            label="Email"
-            placeholder="Enter your email"
-            name="email"
-            control={control}
-            errors={errors}
-          />
-        </p>
-        <p>
-          <Input
-            type="text"
-            label="Password"
-            placeholder="Enter your password"
-            name="password"
-            control={control}
-            errors={errors}
-          />
-        </p>
-        <Button type="submit" label="Sign up" />
+    <div className={styles['container']}>
+      <h1 className={styles['title']}>
+        Create a free account to save your preferences
+      </h1>
+      <form onSubmit={handleFormSubmit} className={styles['form']}>
+        <Input
+          type="text"
+          label="Name"
+          placeholder="Please write your name"
+          name="fullName"
+          control={control}
+          errors={errors}
+        />
+        <Input
+          type="text"
+          label="E-mail"
+          placeholder="Enter your email"
+          name="email"
+          control={control}
+          errors={errors}
+        />
+        <Input
+          type="password"
+          label="Password"
+          name="password"
+          control={control}
+          errors={errors}
+        />
+        <Button
+          type="submit"
+          label="Sign up"
+          className={styles['submit'] as string}
```

Comment:
I saw Farid talking about "as string" in the [cp-13](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/44#discussion_r1304205212) pr